src/crispy/escape.h

```diff
@@ -84,91 +84,91 @@ inline std::string unescape(std::string_view escapedText)
         Hex1,
         Hex2
     };
-    State state = State::Text;
+    state stateText = state::Text;
```

Comment:
i'm not sure `stateText` is the best name for this.
renaming the enum class to `state_type` might work around this name conflict, or the variable `state` to `currentState` instead.